### PR TITLE
[Staging] Profile Detail

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Created the profile detail cover.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

Screenshot_2019-04-05 Takwimu profile Kenya

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/takwimu_ui/src/components/Navigation/index.js [hunk 1]
```diff
@@ -72,10 +76,17 @@ class Navigation extends React.Component {
...
+
     return () => {
-      this.setState(prevState => ({
-        openDrawer: prevState.openDrawer !== drawer ? drawer : null
-      }));
+      const { width } = this.props;
+      this.setState({
+        isMobileDrawerOpen: isWidthUp('md', width)
+          ? false
+          : newOpenDrawer !== null || isMobileDrawerOpen,
```
Comment (kilemensi) [inline takwimu/takwimu_ui/src/components/Navigation/index.js]: Why not something like:
```javascript
const newOpenDrawer = openDrawer !== drawer ? drawer : null;
const hasDrawer = newOpenDrawer != null || isMobileDrawerOpen

...

  isMobileDrawerOpen: !isWidthUp('md', width) && hasDrawer;

```

File: takwimu/takwimu_ui/src/components/Navigation/index.js [hunk 2]
```diff
@@ -72,10 +76,17 @@ class Navigation extends React.Component {
...
+        openDrawer: openDrawer !== drawer ? drawer : null
```
Comment (kilemensi) [inline takwimu/takwimu_ui/src/components/Navigation/index.js]: Isn't this line exactly like `newOpenDrawer` declaration above? Why not `openDrawer: newOpenDrawer`?

File: takwimu/takwimu_ui/src/components/ProfileDetail/index.js [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+    alignItems: 'flex-start',
+    justifyContent: 'space-between',
+    [theme.breakpoints.up('md')]: {
+      position: 'absolute',
+      width: '374px',
+      top: '99px',
+      left: '99px',
+      zIndex: '1',
+      border: 'solid 0.063rem rgba(0, 0, 0, 0.19)',
+      borderRadius: '0 0 1.063rem 1.063rem'
```
Comment (kilemensi) [inline takwimu/takwimu_ui/src/components/ProfileDetail/index.js]: Mixing `px` and `rem` = :bomb:

File: takwimu/takwimu_ui/src/components/Navigation/index.js (hunk 2 above)
Comment (karimkawambwa) [reply takwimu/takwimu_ui/src/components/Navigation/index.js]: good catch

**Pull request merged**